### PR TITLE
images: limit nodejs-devel to Fedora

### PR DESCRIPTION
**Pull request by ptoscano**
It seems that nodejs-devel does not exist on CentOS 9 Stream;
considering that it is strictly needed only on Fedora (according to its
guidelines), limit it to it.

 * [x] image-refresh fedora-rawhide

### CODE REVIEW

File: images/scripts/lib/build-deps.sh [hunk 1]
```diff
@@ -41,4 +41,12 @@ case "$OS_VER" in
     *) ;;
 esac
 
+# pull nodejs-devel on Fedora for compliance with the guidelines on using nodejs modules:
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Node.js/#_buildrequires
+# pull it also on RHEL 8 & derivatives, as it exists there; it does not exist in newer RHELs
+case "$OS_VER" in
+    fedora*|rhel*8|centos*8) EXTRA_DEPS="$EXTRA_DEPS nodejs-devel" ;;
```
Comment (martinpitt) [inline images/scripts/lib/build-deps.sh:48]: Let's please just limit it to Fedora then. I don't see a need to install it on RHEL 8.

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Review by martinpitt — DISMISSED** [review]
Thanks!

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-11.
Log: https://logs.cockpit-project.org/logs/image-refresh-3361-20220510-071212/

**Comment by cockpituous** [comment]
image-refresh fedora-rawhide done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-rawhide-20220510-074040

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**